content/chainguard/chainguard-enforce/installation/preflight-checklist.md

```diff
@@ -0,0 +1,112 @@
+- [X] Most installation methods require you to have `chainctl` installed. You can achieve this by following our [How to Install `chainctl` tutorial](https://edu.chainguard.dev/chainguard/chainguard-enforce/how-to-install-chainctl/).
+- [x] Some installation methods require you to have a special tool like [Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli#install-terraform) or [Helm](https://helm.sh/docs/intro/install/) installed on your computer. 
+- [x] The installation methods that involve Terraform also require you to [set up a cloud account association](/chainguard/chainguard-enforce/cloud-account-associations/) between Chainguard Enforce and your GCP or AWS account.
+- [x] If you're installing the Chainguard Enforce Agent, it's important to know whether the resources you're installing it onto are public or private as the installation methods differ slightly for each.
+
+Not every installation method requires each prerequisite in this list. The reason for this is that we provide two different connection methods for Chainguard Enforce. The method each cluster uses to connect to Enforce is determined at installation, and there are multiple installation procedures for both connection methods which likewise have their own unique prerequisites.
+
+We currently provide two ways to connect your clusters to Chainguard Enforce: with the [Chainguard Enforce Agent](/chainguard/chainguard-enforce/enforce-overview/#the-chainguard-enforce-agent) and through [Agentless connections](/chainguard/chainguard-enforce/how-to-connect-kubernetes-clusters/#agentless-connections).
+
+
+## Agent or Agentless?
+
+An important decision you need to make before installation is whether you want your resources to connect to Chainguard Enforce using the Chainguard Agent or Agentless connections.
+
+A software agent is a program that performs certain actions on behalf of a user. A software agent has the authority to make decisions on its own and doesn’t necessarily require user input. Accordingly, the Chainguard Enforce Agent is able to analyze a given cluster and enforce any policies you’ve applied to the cluster without manual human oversight.
+
+Instead of installing the Agent directly on your cluster, you can allow a remote Agent — managed by Chainguard — to access your cloud account resources on your behalf. A SaaS offering of Enforce, Agentless connections can be useful since they don’t consume any cluster resources. Because they’re managed by Chainguard, issues can often be addressed more quickly than clusters using agentful connections. Additionally, Agentless connections allow for read-only access to your clusters, which some users may find desirable. Be aware that Agentless connections are limited to public GKE and EKS clusters and Google Cloud Run, AWS ECS, and AWS AppRunner projects.
+
+The following table outlines some of the key differences between the Agent and Agentless connections.
+
```

Comment:
What do you think of making the left column bold as they're labels, and title casing `Yes` and `No` for consistency?